src/doodle/doodleController.js

```diff
@@ -71,19 +98,22 @@ export class DoodleController {
   }
 
   render() {
-    const setLines = newLines => {
-      this.lines = newLines;
+    const setLines = lines => {
+      this.newLines = lines;
     };
     render(
-      <DoodleCanvas
-        attachedElement={this._container}
-        size={this._size}
-        tool={this._tool}
-        active={this._doodleable}
-        lines={this._lines}
-        setLines={setLines}
-      />,
-      document.body
+      <Fragment>
+        <DoodleCanvas
+          attachedElement={this._container}
+          size={this._size}
+          tool={this._tool}
+          active={this._doodleable}
+          lines={this.newLines}
+          setLines={setLines}
+        />
+        <DisplayCanvas lines={this.savedLines} container={this._container} />
```

Comment:
Rendering DisplayCanvas before DoodleCanvas should let them both have the same zIndex for duplicate code removal.